### ISSUE TITLE
Grey Space when Resizing with Spectacle 

### ISSUE THREAD

**Issue by jpryda** [sequelpro/sequelpro#3091]
When resizing a window using [Spectacle](https://www.spectacleapp.com/) shortcuts, unreclaimable grey space is produced which squashes the UI:

![screenshot 2018-06-20 10 50 51](https://user-images.githubusercontent.com/7294160/41666285-27cf794a-7478-11e8-8967-3850f23bdace.png)

**Comment by Muffinman** [comment]
I also get this when resizing with [moom](https://manytricks.com/moom/).

**Comment by domluther** [comment]
Yup, bettertouchtool and the same issue here. If I increase the size normally, there's no problem.

If I use my "Maximise Window" shortcut and the window is not touching the top, it  maximises, looks OK briefly and then goes wrong.
If the window is touching the top, it looks wrong immediately.

**Comment by tweekmonster** [comment]
This also happens when disconnecting a monitor from my laptop.

**Comment by dmoagx** [comment]
@tweekmonster https://github.com/sequelpro/sequelpro/issues/3091#issuecomment-398902155

**Comment by dhcmega** [comment]
Hi, I'm still getting this error when resizing with spectacle. I'm using Build 5446 (97c1b857). Should I use a different binary?

**Comment by bladepop** [comment]
Any updates?
